GitTkachenko-BaceControl/ControlWork2/ControlWork2/Program.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ControlWork2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string input = "Kharkiv=1431000,Kiev=2804000,Las Vegas=603400";
+
+            try
+            {
+                var citys = ParseStringAndAddObjectToList(input);
+
+                var cityWithMostPopulated = GetCityWithMostPopulated(citys);
+                var cityWithLongestName = GetCityWithLongestName(citys);
+
+                Console.WriteLine($"Most populated: {cityWithMostPopulated.Name}" +
+                                $"({cityWithMostPopulated.Population} people)");
+                Console.WriteLine($"Longest name: {cityWithLongestName.Name}" +
+                                $"({cityWithLongestName.Name.Length} letters)");
+            }
+            catch(Exception e)
+            {
+                Console.WriteLine(e.Message);
+                Console.ReadLine();
+            }
+
+
+        }
+
+        private static List<City> ParseStringAndAddObjectToList(string input)
+        {
+            if(string.IsNullOrEmpty(input))
+            {
+                throw new Exception($"Please check the input string. It's empty." + Environment.NewLine +
+                                    $"Type of input string must be as: \"Kharkiv=1431000,Kiev=2804000,Las Vegas=603400\"");
+            }
+
+            var citys = new List<City>();
+            var arrayCityes = input.Split(',');
+
+            foreach (var i in arrayCityes)
```

Comment:
'var i' -> 'string city'. The name of the variable is not a counter here, so it should be meaningful, and it's type isn't obvious, so it should be used instead of 'var'.